tark/tark/templates/documentation.html

```diff
@@ -1,11 +1,91 @@
+<head>
+	<style>
+	<!---------------------------------------------------------------------------------->
+	<!--------------------------------Add CSS for collapsible------------------>
+	<!---------------------------------------------------------------------------------->
+	.collapsible {
```

Comment:
It's best practice to have CSS in a separate file (e.g. a file inside `static/css` directory)